packages/app/src/components/severity-indicator-tile/components/timeline/components/timeline-bar-part.tsx

```diff
@@ -0,0 +1,39 @@
+import { transparentize } from 'polished';
+import { ReactNode } from 'react';
+import styled from 'styled-components';
+import { Box } from '~/components/base';
+
+interface TimelineBarPartsProps {
+  children: ReactNode;
+  color: string;
+  size: number;
+  width: string;
+  isFirst?: boolean;
+  isLast?: boolean;
+}
+
+export const TimelineBarPart = ({ children, color, size = 10, width, isFirst = false, isLast = false }: TimelineBarPartsProps) => {
+  const borderRadius = isFirst ? `${size / 2}px 0 0 ${size / 2}px` : null;
+
+  return (
+    <Box
+      as="li"
+      alignItems="center"
+      backgroundColor={transparentize(0.7, color)}
```

Comment:
this would make it also 70% transparent and would get rid of the transparentize functionality
backgroundColor={`${color}B3`)}
```suggestion
      backgroundColor={`${color}B3`)}
```

Reply:
While I also looked into this when working on this feature, I argued to not go with solution as it might not as clear as using a method with a straight forward name such as 'transparentize'. Using an 8-digit HEX color will require you to look up the HEX code value for the opacity, while something like `transparentize` is already available in the project and makes this code more readable in my opinion.

The tricky part about `transparentize` is that it actually deducts an opacity of `0.7`, leaving an actual opacity of `0.3` on the color. 🤔

Reply:
Good point, that makes sense. The inverted opacity would also be a no-go for me. I think we can discuss this or an alternative solution at another time and another place.